src/components/SuggestionsList/SuggestionsListItem.css

```diff
@@ -13,3 +13,29 @@
 .y-suggestions-list-item--selected {
   background-color: $suggestionsListItemBackgroundColor__keyboardNavigate;
 }
+
+/* Windows High Contrast support */
+@media screen and (-ms-high-contrast: active) {
+  /* Selected items should use theme color for background... */
+  .y-suggestions-list-item:hover,
+  .y-suggestions-list-item--selected {
+    background: Highlight;
+  }
+
+  /* and text. Need to force this on nested spans to override
+     the default regular non-selected text color there. */
+  .y-suggestions-list-item:hover *,
```

Comment:
can we use `>` child selectors here to avoid the `*` looking for all ancestors having the class+hover?

Reply:
No, unfortunately, as we can't make assumptions about the structure under this item. Turns out the text we specifically want to target here is a couple of divs deep, since this suggestion item contains a media object which contains a fixed grid which then contains the item we want.